pkg/daemon/ceph/osd/volume.go

```diff
@@ -604,6 +606,12 @@ func getCephVolumeLVMOSDs(context *clusterd.Context, clusterName string, cephfsi
 			if osd.Type == "journal" {
 				store = "filestore"
 			}
+
+			// If no lv is specified let's take the one we discovered
+			if lv == "" {
+				lv = osd.Path
```

Comment:
Do we only expect one osd to be found in the `osdInfo` list? It's odd that we're looping over all osds to create a single `OSDInfo` struct below outside the for loop.